ReactionHandler.js

```diff
@@ -64,6 +70,65 @@ class ReactionHandler {
     if (type === `ADD`) Member.roles.add(Role);
     if (type === `REMOVE`) Member.roles.remove(Role);
   }
+
+  async handleStarReaction(Reaction) {
+    await Reaction.fetch();
+    let starboardChannelId;
+    let isAlreadyOnStarboard = false;
+
+    if (!starConfig.ignoreChannels.includes(Reaction.message.channel.name)
+          && Reaction.count === starConfig.starThreshold) {      
+      // Message not in a black-listed channel and meets star threshold
+      
+      // Construct the starboard message
+      const starMessage = new Discord.MessageEmbed()
```

Comment:
Need to make this so it can also handle video embeds.

Reply:
Video embeds are currently not supported. Added some logic/helpful text to resolve.